examples/cluster/plot_digits_linkage.py

```diff
@@ -65,7 +70,7 @@ def plot_clustering(X_red, labels, title=None):
             X_red[i, 1],
             str(y[i]),
             color=plt.cm.nipy_spectral(labels[i] / 10.0),
-            fontdict={"weight": "bold", "size": 9},
+            fontdict={"weight": "bold", "size": 10},
```

Comment:
The font size change does not seem strictly necessary. I would probably leave it like it was before to make the PR more concise.

What do you think?

Reply:
Okie I have changed it back